src/test/java/com/bandwidth/sdk/messaging/MessagingClientTest.java

```diff
@@ -200,4 +200,21 @@ public void testDeleteMedia() {
 
         client.deleteMedia("image.jpg");
     }
+
+    @Test
+    public void testMultipleClientsWithDifferentBaseUrls() {
```

Comment:
I guess it's okay for bug regression purposes... but we wouldn't normally unit test something like this because it'd be overkill to write a unit test for every single class field with state. It's basically a design rule for the most part that class fields with state should not be static.

Reply:
I'll remember that for the future